### PR TITLE
Git inspections

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

How it may look like:
image

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^8.0",
+        "cypresslab/gitelephant": "^4.5",
```
Comment (vjik) [inline composer.json:26]: This package use GNU Lesser General Public License v3.0, seems, we don't can use it.

Comment (xepozz) [reply composer.json:26]: Why not?

Comment (vjik) [reply composer.json:26]: > Why not?

I dont sure, but seems we don't can distribute package with BSD-3-Clause license if use packages with LGPL license.

Comment (samdark) [reply composer.json:26]: Yes, better to avoid GPL if possible.

**Pull request merged**